src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -130,4 +131,15 @@ public ResponseEntity<Map<CreationEnum, List<?>>> createMediaAccountsBulk(
         @RequestHeader("x-issuer-id") String issuerId, @RequestPart MultipartFile mediaList) {
         return ResponseEntity.ok(accountService.uploadMediaFromCsv(mediaList, issuerId));
     }
+
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Account with provenance id {provenanceUserId} has been verified"),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE),
+        @ApiResponse(code = 404, message = "User with supplied provenance id: {provenanceUserId} could not be found"),
+    })
+    @ApiOperation("Update the last verified date for a media user")
+    @PutMapping("/media/verification/{provenanceUserId}")
```

Comment:
I know the whole process is for updating the verification date for media users only, but this controller and the service class are generic. You are passing in a provenance ID and setting the last verified date for the user with the provenance ID, regardless of the user type here. I wonder if we should rename the endpoint path to be just /account/verification/{provenanceId} without mentioning the word 'media' here? This also applies to the method names/descriptions in both AccountController and AccountService. Happy to discuss :)

Reply:
Another thing is I will be adding another endpoint (or refactor yours to take in a generic field/value pair, not decided yet) in my future ticket for last sign-in date. That one applies to both AAD admin and all IDAM users, so I can't specially both user types in the endpoint path. So just thinking of keeping the naming consistent for both.

Reply:
Makes sense. Will refactor to generic then when you do the Admin inactive accounts part if you want to add any more params to endpoint to workout the type of user you can. :)